core/domain/recommendations_jobs_one_off.py

```diff
@@ -54,29 +54,27 @@ def map(item):
         # Note: This is needed because the exp_summaries_dict is sometimes
         # different from the summaries in the datastore, especially when
         # new explorations are added.
-        if exp_summary_id not in exp_summaries_dict:
```

Comment:
Same thing. Leave the original code as-is.

Reply:
I couldn't seem to test a condition when `exp_summary_id` is not in `exp_summaries_dict`. I tried by creating a new exploration as suggested in the comment but still couldn't test it. Any suggestions?

Reply:
You can mock get_non_private_exploration_summaries() by using self.swap.

Reply:
Done